packages/react-i18n/src/constants/index.ts

```diff
@@ -3,9 +3,17 @@ export enum DateStyle {
   Short = 'Short',
   Humanize = 'Humanize',
   Time = 'Time',
+  LongDateTime = 'LongDateTime',
 }
 
 export const dateStyle = {
+  [DateStyle.LongDateTime]: {
```

Comment:
What if I keep the weekday in there, and we name it `DateTime`. So that is in between long and short and maybe resonates what I am trying to do here. For my purposes I can just remove the weekday in the format I want?

As mentioned below for the `README.md`, I can include text that it uses that translation key and what the type of results are.